### PR TITLE
Divyanshu | Test | Version 3 - Get all voucher page - Bugs

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cwfaavg

### CODE REVIEW

File: apps/web-giddh/src/assets/locale/all-items/mr.json [hunk 1]
```diff
@@ -213,7 +213,7 @@
                 },
                 {
                     "label": "पावती",
-                    "link": "/pages/reports/receipt",
+                    "link": "/pages/vouchers/preview/receipt/list",
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/assets/locale/all-items/mr.json:227]: Write for both versions

File: apps/web-giddh/src/app/vouchers/preview/preview.component.html [hunk 2]
```diff
@@ -124,7 +124,7 @@ <h3 class="font-16 fw-bold">{{ selectedInvoice?.voucherNumber }}</h3>
                         <p>{{ selectedInvoice?.account?.customerName || selectedInvoice?.account?.name }}</p>
                     </div>
                     <div class="d-flex column-gap1 flex-wrap row-gap1">
-                        <button mat-stroked-button color="primary" (click)="toggleActivityHistoryAsidePane()" *ngIf="!(invoiceType.isEstimateInvoice || invoiceType.isProformaInvoice)">
+                        <button mat-stroked-button color="primary" (click)="toggleActivityHistoryAsidePane()" *ngIf="!(invoiceType.isEstimateInvoice || invoiceType.isProformaInvoice) || true">
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/vouchers/preview/preview.component.html]:
```suggestion
                        <button mat-stroked-button color="primary" (click)="toggleActivityHistoryAsidePane()" *ngIf="!(invoiceType.isEstimateInvoice || invoiceType.isProformaInvoice)">
```

File: apps/web-giddh/src/app/vouchers/preview/preview.component.ts [hunk 3]
```diff
@@ -178,6 +180,9 @@ export class VouchersPreviewComponent implements OnInit, OnDestroy {
     private getAllApiCallCount: number = 0;
     /** Holds current route query parameters */
     public queryParams: any = {};
+    public voucherVersions: ProformaVersionItem[] = [];
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/vouchers/preview/preview.component.ts:183]: JS Doc

**Review by ankitdubeyoffice — APPROVED** [review]

File: apps/web-giddh/src/app/vouchers/preview/preview.component.html [hunk 4]
```diff
@@ -297,6 +296,34 @@ <h3 class="font-16 fw-bold">{{ selectedInvoice?.voucherNumber }}</h3>
                         <a href="javascript:;" (click)="openEmailSendDialog()">{{ localeData?.send_invoice }}</a>
                     </div>
                 </div>
+                <div class="d-flex" *ngIf="invoiceType.isEstimateInvoice || invoiceType.isProformaInvoice">
+                    <div>
```
Comment (ankitdubeyoffice) [inline apps/web-giddh/src/app/vouchers/preview/preview.component.html:300]:
```suggestion
```

**Review by ankitdubeyoffice — CHANGES_REQUESTED** [review]

**Review by ankitdubeyoffice — APPROVED** [review]

**Pull request merged**